qlkube/src/decorateSubscription.js

```diff
@@ -22,6 +22,199 @@ function getSubQueryList(queryName) {
   else return `${queryName}List`;
 }
 
+function getSublabels(targetType) {
+  return wrappers
+    .map(wtype => {
+      if (wtype['parents'].includes(targetType)) return wtype['type'];
+    })
+    .filter(s => {
+      return s;
+    });
```

Comment:
```suggestion
    .map(({ parents, type }) => parents.includes(targetType) && type)
    .filter(s =>s);
```

Reply:
So, I think that even `if (wtype['parents'].includes(targetType)) return wtype['fieldWrapper'];` could be changed at the same way

Reply:
yes, I wrote it once but it can clearly be applied everywhere. Just test it behaves correctly because you can never know what will happen with js 🥴